src/utils/Cortex.js

```diff
@@ -35,7 +35,16 @@ export function cortexFetch(input, init) {
     return mockFetch(input, requestInit);
   }
 
-  return fetch(`${Config.cortexApi.path + input}`, requestInit)
+  let queryFormat = '';
+  if (input && (Config.formatQueryParameter.standardlinks || Config.formatQueryParameter.noself || Config.formatQueryParameter.nodatalinks)) {
+    const queryCharacter = input.includes('?') ? '&' : '?';
+    const standardlinks = Config.formatQueryParameter.standardlinks ? 'standardlinks,' : '';
+    const noself = Config.formatQueryParameter.noself ? 'zoom.noself,' : '';
+    const nodatalinks = Config.formatQueryParameter.nodatalinks ? 'zoom.nodatalinks' : '';
+    queryFormat = `${queryCharacter}format=${standardlinks}${noself}${nodatalinks}`;
+  }
+
+  return fetch(`${Config.cortexApi.path + input + queryFormat}`, requestInit)
```

Comment:
to avoide issues use `${var1}${var2}` to concatenete strings: `${Config.cortexApi.path}${input}${queryFormat}`